### PR TITLE
UI/request approval

### PR DESCRIPTION
**Pull request by luisgov**
This PR adds the static UI for the CCPO approval step.
`http://localhost:8000/request_approval`

Next steps will be to abstract the fields and leverage the TextInput module and make the form submit.

screencapture-localhost-8000-request_approval-2018-09-05-13_37_47

### CODE REVIEW

**Comment by andrewdds** [comment]
@luisgov looks good, just a couple details about the comments section:
- The "Approval Notes" title can be bumped up one heading size.
- Within the blue box, the field label seems redundant. Could you make the "Comments to Requestor" heading be the main label, and the rest just be the field description.

File: templates/request_approval.html [hunk 1]
```diff
@@ -68,56 +69,114 @@ <h2 class='h3'>Approval Notes</h2>
...
+
+
     </section>
 
     <section class='action-group'>
-      <a href='/' class='usa-button usa-button-big'>Approve Request</a>
-      <a href='/' class='usa-button usa-button-big usa-button-secondary'>Deny Request</a>
-      <a href='/' class='icon-link'>
+      <a href='#' class='usa-button usa-button-big'>Approve Request</a>
+      <a href='#' class='usa-button usa-button-big usa-button-secondary'>Mark as Changes Requested</a>
```
Comment (andrewdds) [inline templates/request_approval.html]: Maybe just "Request Changes"
This phrase reads a little funny

Comment (luisgov) [reply templates/request_approval.html]: 👍

**Comment by hardikdds** [comment]
@luisgov   Looking good on the approval part.  


FYI I think the internal comments should probably be a separate PR on its own I suppose?  That one will need an edit and save function per this story:  https://www.pivotaltracker.com/story/show/160141410 .

**Comment by andrewdds** [comment]
@luisgov I was suggesting the paragraph at the top of the info alert become the field description, below the bold field label

**Pull request merged**